lib/elixir/lib/kernel.ex

```diff
@@ -3143,6 +3143,12 @@ defmodule Kernel do
 
       Enum.map(List.flatten([1, [2], 3]), fn x -> x * 2 end)
 
+  Assignment is also possible with the pipe operator.  In the example below we
+  assign flatten list to the variable `flattened_list`:
+
+      iex> flattened_list = [1, [2], 3] |> List.flatten()
+      [1, 2, 3]
+
```

Comment:
I am a little bit skeptical about adding this change here. The result of every operator in the language can be assigned to a variable. This is a general language property and not specific to how this operator is used. Does it mean we need to add a similar sentence to every operator? Thoughts?